src/Project.hs

```diff
@@ -253,11 +238,10 @@ classifyPullRequest pr = case approvedBy pr of
   Just _  -> case integrationStatus pr of
```

Comment:
If a pull request can only be integrated if it was approved, is this another thing we would want to express in the ADTs?

Reply:
Yes, I would like to do this, but not as part of this PR, as it would be a change of a similar size, again touching all the same places.